### PR TITLE
add contributing guidelines

### LINKED ISSUE #43: Add documentation how to contribute to the project
Can someone please add some documentation how to contribute to the project? What's the workflow when you want to add a new command or extend an existing one and want to test whether it works? 

For example, how would go about implementing ticket #42 ?

### PR DESCRIPTION
**Pull request by ltalirz**
fixes #43

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+Both the [dokku ansible role](https://github.com/dokku/ansible-dokku) and [dokku](https://github.com/dokku/dokku) itself are open source projects and contributions are highly welcome!
```
Comment (josegonzalez) [inline CONTRIBUTING.md:3]: Can we reference the org's CONTRIBUTING.md?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   In order to update it, run `make generate`.
+
+## Setting up a test environment
+
+This role is tested using [molecule](https://molecule.readthedocs.io/en/latest/).
+Setting up a test environment involves the following steps:
+
+ * Install [docker](https://www.docker.com/)
+ * Install [python](https://www.python.org/)
+ * Run `pip install -r  requirements.txt`
```
Comment (josegonzalez) [inline CONTRIBUTING.md:19]: Should we run tests in a virtualenv?

Comment (ltalirz) [reply CONTRIBUTING.md:19]: I certainly use virtual environments for this; happy to mention it as an optional step.
One can also switch to something like tox but in my view it also makes things more complicated to understand.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ltalirz) [reply CONTRIBUTING.md:3]: Sure, you mean this link https://github.com/dokku/dokku/blob/master/CONTRIBUTING.md , correct?

Comment (josegonzalez) [reply CONTRIBUTING.md:3]: Nah we have a global org thing here: https://github.com/dokku/.github

File: CONTRIBUTING.md (hunk 2 above)
Comment (josegonzalez) [reply CONTRIBUTING.md:19]: Hmmm up to you. I basically want to make sure we can cleanly run things without ruining someone's laptop :)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Contributing
 
-Both the [dokku ansible role](https://github.com/dokku/ansible-dokku) and [dokku](https://github.com/dokku/dokku) itself are open source projects and contributions are highly welcome!
+
```
Comment (josegonzalez) [inline CONTRIBUTING.md:3]: Remove the extra newline :)

Comment (ltalirz) [reply CONTRIBUTING.md:3]:
```suggestion
```

**Review by josegonzalez — APPROVED** [review]

**Comment by ltalirz** [comment]
Sorry, I realized you already approved this. Will merge and make a new release.

**Pull request merged**